tests/common/reboot.py

```diff
@@ -277,6 +281,12 @@ def reboot(duthost, localhost, reboot_type='cold', delay=10,
         if not ret:
             raise Exception('warmboot-finalizer service timeout on DUT {}'.format(hostname))
 
+    # Verify if the temporary file created in tmpfs is deleted after reboot, to determine a
+    # successful reboot
+    file_check = duthost.command('find /dev/shm/test_reboot', module_ignore_errors=True)['rc']
```

Comment:
Instead of using your own way of checking for file, you could use preexisting method that comes with Ansible:

```
file_stat = duthost.stat(path="/dev/shm/test_reboot")
pytest_assert(not file_stat["stat"]["exists"], "Test file was found on DUT after reboot")
```

Reply:
Examples: https://github.com/sonic-net/sonic-mgmt/blob/80415ad3f79c4851091a779bbe2f06c776c929af/tests/common/helpers/backend_acl.py#L26

https://github.com/sonic-net/sonic-mgmt/blob/80415ad3f79c4851091a779bbe2f06c776c929af/tests/bgp/test_bgp_sentinel.py#L305

Reply:
Modified the PR to use the existing method. Thank you!